client/routes/privacy.js

```diff
@@ -3,7 +3,206 @@ import React from 'react';
 function Privacy() {
     return (
         <div>
```

Comment:
Probably want to give this div a margin of some sort so the text isn't right against the edge of the screen.

Reply:
fixed